qmq-common/src/test/java/qunar/tc/qmq/utils/utils/RetrySubjectUtilsTest.java

```diff
@@ -0,0 +1,46 @@
+package qunar.tc.qmq.utils.utils;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+import static qunar.tc.qmq.utils.RetrySubjectUtils.*;
+
+public class RetrySubjectUtilsTest {
+
+    @Test
+    public void testGetRealSubject() {
+        assertNull(getRealSubject(null));
+
+        assertEquals("%RETRY", getRealSubject("%RETRY"));
+        assertEquals("foo", getRealSubject("%DEAD_RETRY%foo%bar"));
+    }
+
+    @Test
+    public void testParseSubjectAndGroup() {
+        assertNull(parseSubjectAndGroup(null));
+        assertNull(parseSubjectAndGroup("%DEAD_RETRY"));
+
+        assertNotNull(parseSubjectAndGroup("%DEAD_RETRY%foo%bar"));
```

Comment:
NotNull is not enough

Reply:
added an assertEquals to check length of return value String[]

Reply:
Why not test the content?

Reply:
Sorry it was returning a memory address. I changed the assert to check for the contents